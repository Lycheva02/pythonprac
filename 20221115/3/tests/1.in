alp = Alpha(c=10, b=12,d=3)
alp.k = 132
print(alp)
try:
    alp.R = 17
except AttributeError:
    print("Molodets")

alq = AlphaQ(c=10, b=12,d=3)
alq.k = 132
print(alq)
try:
    alq.R = 17
except AttributeError:
    print("Molodets")