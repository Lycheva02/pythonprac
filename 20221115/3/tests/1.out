b: 12, c: 10, d: 3, k: 132
Molodets
b: 12, c: 10, d: 3, k: 132
Molodets
